test/schema/default_config_test.go

```diff
@@ -73,7 +73,8 @@ var defaultVectorIndexConfig = map[string]interface{}{
 }
 
 var defaultReplicationConfig = &models.ReplicationConfig{
-	Factor: 1,
+	Factor:                           1,
+	ObjectDeletionConflictResolution: models.ReplicationConfigObjectDeletionConflictResolutionPermanentDeletion,
```

Comment:
this was merged prematurely and we will most likely change the name again (ObjectDeletionConflictResolution) - do we need to set it in this PR?

Reply:
Without this field [one of the integration tests is failing](https://github.com/weaviate/weaviate-go-client/actions/runs/11313124509/job/31461500050#step:5:1668). So it's either setting the field or t.Skip-ing the test.

I'd do the former -- once that field is renamed, the next person to do a PR to this repo will be reminded to update it by the compiler.

Reply:
Could you use the latest version from weaviate:stable/v1.26? We just renamed it there